content/quiz_events.md

```diff
@@ -4,21 +4,139 @@
+- The quizzer upon whom a foul is called becomes ineligible to answer that numbered question, including the A and B questions.
+- Three fouls by a quizzer eliminate that quizzer from the quiz.  The quizzer must leave the platform immediately.  His/her seat will continue to be ineligible to jump for the entire numbered question including A and B.
+
+The following are fouls:
+- Team members (quizzers, coaches) communicating with each other, other teams, or with the audience between the time the question type is introduced, and points are awarded. 
+     - Communication can include non-verbal communication.
+- The use of the hands to assist in jumping.  Hands may not touch any part of the chair or the floor from the time "Question" is called until the quizmaster recognizes the first one up.
+- When electronic equipment is used, if any light goes on after the quizmaster calls "Question" and before they begin to discernibly read the question.
+    - Discernibly includes the quizmaster's mouth forming a word shape.
+- During a Reference question or Quote question, when electronic equipment is used, if any light goes on after the quizmaster calls “Question” and before they begin to discernibly read the chapter number.
+- Starting to answer before being recognized by the Quizmaster.
+- Having more than the designated team members, one coach, and an assistant coach on the quiz platform during a time-out.
+- Challenging after conferring with anyone
+- If a captain or co-captain confers with his/her coach or other quizzers prior to issuing a challenge.  In addition, the challenge will not be allowed.
+- Infraction of the rules, in spirit as well as letter, as discerned by the quiz officials.
+
+Individual Fouls
+: A foul awarded to one or more individual quizzers.
+
+Team Fouls
+: The sum of individual fouls.
 
 ## Time-Outs
 
     Set the timeout duration to 60.
 
+- Each team is allowed two, one-minute time-outs.  Only one of those time-outs may be taken during questions 18-20. 
+- Captains, co-captains or coaches may call for a time-out. 
+- Only the coach, assistant coach, and substitute may converse with the team during a time-out.
+- Time-outs may be called prior to bonus questions. 
+
 ## Substitutions
+- Substitutions may be made only during the time-outs.
+- A quizzer being removed from a quiz must stay out for at least three numbered questions including A and B before re-entering the quiz.
+- Quizzers can be substituted for any chair.
+- Quizzed-Out Quizzers:
+    - May remain in the quiz to answer bonus questions.
+        - Substituting for them later will require a time-out.
+    - If subbed out, may not return, for any reason.
+- When a quizzer quizzes-out, err-out, or fouls-out, and leaves the platform as required, a substitute may take their place:
+    - Immediately
+    - Without a time-out
+    - Even if the quizzer has been subbed-out less than three questions ago
 
 ## Scoring
```

Comment:
### SCORING
**INDIVIDUAL POINTS**

_**Points Earned**_
- +20 Points for every correct question & toss-up question
- +10 Points for every quiz out without error

**_Points Deducted_**
- -10 Points for 2nd and subsequent personal errors
     - Points are not deducted for errors on bonus questions
     - Points are not deducted for errors during overtime
- -10 Points for 3rd personal foul


**TEAM POINTS**
All point earned or deducted by an individual are to be counted towards the team’s points.

_**Points Earned**_
- +20 Points for team arriving on time. 
     - Teams that arrive late due to quizzing in another room are excused from the forfeiture of these points. If a single quizzer is late, the coach can decide to keep the +20 points by keeping the late quizzer out the whole quiz, or forfeit the points by subbing the quizzer in after question number 1. This decision must occur before the quiz has been started. 
- +10 Points for every 3rd and subsequent correct answer given by a unique quizzer on the team
- +20 Points for every correct bonus question before question number 17
     - In 2 Team Quizzes, this scoring rule does not apply 
- +10 Points for every correct bonus question after and including question number 17
     - In 2 Team Quizzes, all correct bonus questions are worth 10 points

_**Points Deducted**_
- -10 Points for every team error starting at error number 3
- -10 Points for every error starting at question number 17
- -10 Points for 4th and subsequent team fouls
- -10 Points for 2nd and subsequent overruled challenges & protests
- -10 Points at the determination of the room officials that a deliberate attempt was made to forfeit a question.